### ISSUE TITLE
Set up Dot & its Logic (Scripts)

### ISSUE THREAD

**Issue by trentwirth** [vilonlab/dot-chaser#5]
Dot is just a White Sphere that sits on (in?) the ground plane.

Technically blocked by #2 and #3

Dot logic:
- changes color when is fixated upon
- moves when you get to close to it (hitbox interaction with Headset)
    - new location is based on walkable play space and the current position of Participant (headset)
- moves a certain number of times (defined before the experiment starts)

**Comment by trentwirth** [comment]
connected to issue #9 , we will record Dot position and set up its behavior in a Dot Class.

Could be one big Dot Class with a DotBehavior and DotDataWriter methods, or we could have separate DotBehavior.cs and DotDataWriter.cs scripts that are both meant to function when attached to the 3D Sphere.

## Desired Dot Behaviors:
- Dot appears within tracking space, randomly somewhere relative to the participant, between `Distance Lower Bound` and `Distance Upper Bound`, where Lower = 0.25m, and Upper = 2m (to start/test with).
- (*Stretch Goal*) Interact with the gaze vectors; when the dot is being looked at, have it change color from White to Green. See [this video](https://www.youtube.com/watch?v=ZoySn7QlMfQ&t=1315s)

## Desired Dot Data Writer
- For now, have it work just like the EyeDataWriter script: Scene is loaded up, and the position of the dot (X, Y, Z) is recorded for every frame.
